nestkernel/archiving_node.cpp

```diff
@@ -275,6 +275,19 @@ nest::Archiving_Node::set_status( const DictionaryDatum& d )
     clear_history();
   }
 
+
+  if ( d->known( names::update_synaptic_elements ) )
+  {
+    const DictionaryDatum synaptic_elements_a =
+      getValue< DictionaryDatum >( d, names::update_synaptic_elements );
+    for ( std::map< Name, SynapticElement >::iterator it =
+            synaptic_elements_map_.begin();
+          it != synaptic_elements_map_.end();
+          ++it )
+    {
+      it->second.set( synaptic_elements_a );
```

Comment:
Just a quick comment to inform the reviewers of an issue here (@sdiazpier is already aware of this):

The current implementation does not enable the user to set different parameter values for different synaptic elements. Usually, though, as in the Butz paper that introduces MSP, neurons will at least have two types of synaptic elements - the axonal and the dendtritic, and these will both have growth curves described by different sets of parameters. So, one must be able to set their parameters separately here too. The fix is quite simple - enable the user to mention which synaptic element is to be updated.